appmap/django.py

```diff
@@ -131,7 +132,18 @@ def __call__(self, request):
             )
             self.recorder.add_event(call_event)
 
-        response = self.get_response(request)
+        try:
+            response = self.get_response(request)
+        except Exception as e:
```

Comment:
You're not using `e`; might as well not bind it and just reraise with plain `raise`.